notebooks/Beta_Visualization.ipynb

```diff
@@ -0,0 +1,729 @@
+{
```

Comment:
<p>At the very beginning of this notebook, add a Set up section, with the same code as found here</p><p>https://github.com/geomstats/geomstats/blob/master/notebooks/04_practical_methods__from_vector_spaces_to_manifolds.ipynb</p><p> </p><p>This will allow to correct the No module named 'geomstats.visualization.beta_distributions'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>